src/core/memory.d

```diff
@@ -165,9 +165,12 @@ struct GC
     static struct Stats
     {
         /// number of used bytes on the GC heap (might only get updated after a collection)
-        size_t usedSize;
+        ulong usedSize;
         /// number of free bytes on the GC heap (might only get updated after a collection)
-        size_t freeSize;
+        ulong freeSize;
+        /// number of bytes freed during collections through program lifetime so
+        /// far (will count same memory multiple times if re-used)
```

Comment:
What's the point of keeping track of this? It should be mentioned at least in the commit message, but probably also in the changelog and the code docs.